src/AppBundle/Migration/Core/FieldHandler/EzImageAsset.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace AppBundle\Migration\Core\FieldHandler;
+
+use eZ\Publish\Core\FieldType\ImageAsset\Value as ImageValue;
+use Kaliop\eZMigrationBundle\API\FieldValueConverterInterface;
+use Kaliop\eZMigrationBundle\Core\FieldHandler\AbstractFieldHandler;
+use Kaliop\eZMigrationBundle\Core\ReferenceResolver\PrefixBasedResolverInterface;
+
+class EzImageAsset extends AbstractFieldHandler implements FieldValueConverterInterface
+{
+    protected $resolver;
+
+    public function __construct(
+        PrefixBasedResolverInterface $resolver
+    ) {
+        $this->resolver = $resolver;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function hashToFieldValue($fieldValue, array $context = array())
+    {
+        return new ImageValue(
```

Comment:
Maybe we should check first if keys 'destinationContentId' and 'alternativeText' exists for $fieldValue and if not then throw exception?

Reply:
updated, but this is the convention used by migration bundle, see
```
\Kaliop\eZMigrationBundle\Core\FieldHandler\EzPage::hashToFieldValue
\Kaliop\eZMigrationBundle\Core\FieldHandler\EzRichText::hashToFieldValue
```